### PR TITLE
[add] footer logic and styles

### PR DESCRIPTION
**Pull request by Way2nnadi**
- we now have a clean footer section

### CODE REVIEW

**Comment by Way2nnadi** [comment]
View of the new footer
screen shot 2018-12-17 at 3 08 38 pm

File: src/components/home/components/Footer.js [hunk 1]
```diff
@@ -1,31 +1,28 @@
...
 
-const { ProductsAvailable } = components;
-const { Content, Footer } = Layout;
-const { CENTER, FLEX, FOOTER, FOOTER_CONTENT, FOOTER_TEXT, LOWER_FOOTER } = FOOTER_STRINGS;
+const { Footer } = Layout;
+const { CONTAINER, DYNAMIC_FOOTER, ROW } = FOOTER_STRINGS;
 
-//This contains the productsavailable component
-class  AppFoot extends React.Component {
+const footers = FOOTER_DATA.map(footer => (
```
Comment (EngrEric) [inline src/components/home/components/Footer.js]: coding is becoming more interesting for me

Comment (Way2nnadi) [reply src/components/home/components/Footer.js]: Yeah, maps are some of the most simple and powerful tools in software development

Comment (joshuaobinna) [reply src/components/home/components/Footer.js]: Wow
We are getting there
footer really looks good

**Pull request merged**